src/components/Tabs/_tests_/Tab.test.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import { Tabs, Tab } from "..";
+import { TabLabel } from "../TabLabel";
+
+describe('<Tab/>', () => {
+    test('renders without crushing', () => {
+        render(<Tab label="Steps"/>);
+    });
+    test('props work', () => {
+        const label = "Calls"
+        const testLabel = "NewCalls"
+        const { getByTestId } = render(<Tab label={label} info={testLabel}></Tab> );
+        const elem = getByTestId('label')
+        expect(getByTestId('label').textContent).toBe(testLabel);
+        
+
+    });
+});
+
+describe('<TabLabel/>', () => {
+    test('renders without crushing', () => {
```

Comment:
the best way to achieve this is by using a snapshot test 😉 

In this test, you are not running any assertion, whilst you should have at least one assertion per test. As essentially you are making sure that the components render properly, I think a snapshot test would be a great scenario